app/assets/src/components/Samples.jsx

```diff
@@ -1891,7 +1893,10 @@ function ProjectInfoHeading({
         {state.selectedProjectId ? project_menu : null}
         {table_download_dropdown}
         {compare_button}
-        {state.selectedProjectId && state.allSamples.length == 0
+        {state.selectedProjectId &&
+        canEditProject(state.selectedProjectId) &&
```

Comment:
would this work even if you didn't pass it in? since canEditProject was already bound to 'this' at the top?

Reply:
It didn't...